### DISCUSSION TITLE
download failure

### DISCUSSION THREAD

**Discussion by mintlong** [aclap-dev/video-downloadhelper#721] [Help and Q&A]
### Your question

downloads keep getting this error message

### Addon & CoApp version and details

{
  "version": "9.0.1.13",
  "target": "mozilla",
  "channel": "stable",
  "page_url": "undef",
  "error": {
    "error": "coapp_failure",
    "details": "Error: Native port disconnected",
    "id": "coapp_failure_3b87b6e1-041b-42b3-b239-6b6f0aa42c9b",
    "downloadable_id": "downloadable_3472141782997982"
  }
}

**Comment by GrampaWildWilly** [comment]
You need to supply the specific URL of the web page on which this happens.  Not the name of a web site.  A specific web page.  If it's happening "everywhere," at least give one example of a specific page where it's happening.
